scopus-import/src/main/java/no/sikt/nva/scopus/conversion/model/cristin/CristinPerson.java

```diff
@@ -24,6 +24,7 @@
 import no.unit.nva.commons.json.JsonSerializable;
 import nva.commons.core.JacocoGenerated;
 
+@SuppressWarnings("PMD.BooleanGetMethodName")
```

Comment:
is this not a simple fix? isSomeValue

Reply:
Maybe, I didn't try here. The code is copied from `cristin-service` and when I refactored just like that there a bunch of tests started failing because of some weirdness with fieldnames hardcoded as strings in a big list somewhere.

There were hundreds of these issues and at some point I just gave up....